<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uiPref</class>
 <widget class="QWidget" name="uiPref">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Helvetica Neue</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Type BDD</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="bddType">
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Helvetica Neue</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Base de donn√©e </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="bddName">
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Helvetica Neue</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>User BDD</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="bddUser">
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Helvetica Neue</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Password BDD</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="bddPass">
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="SaveButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Images.qrc">
       <normaloff>:/new/MenuInterface/Ressouces/cloud-minus@2x.png</normaloff>:/new/MenuInterface/Ressouces/cloud-minus@2x.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>35</width>
       <height>35</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Images.qrc"/>
 </resources>
 <connections/>
</ui>
